INSTALL.md

```diff
@@ -1,40 +1,53 @@
-# INSTALLATION INSTRUCTIONS FOR GREENBONE VULNERABILITY MANAGER
+# Installation Instructions for Greenbone Vulnerability Manager Daemon
 
 Please note: The reference system used by most of the developers is Debian
-GNU/Linux 'Buster' 10. The build might fail on any other system. Also, it is
-necessary to install dependent development packages.
+stable. The build might fail on any other system. Also, it is necessary to
+install dependent development packages.
 
-## Prerequisites for Greenbone Vulnerability Manager
+## Prerequisites for Greenbone Vulnerability Manager Daemon
 
 Prerequisites:
-* GCC (Debian package: gcc)
-* cmake >= 3.0 (Debian package: cmake)
-* cJSON >= 1.7.14 (Debian package: libcjson-dev)
-* glib-2.0 >= 2.42 (Debian package: libglib2.0-dev)
-* gnutls >= 3.2.15 (Debian package: libgnutls28-dev)
-* libgvm_base, libgvm_util, libgvm_osp, libgvm_gmp >= 20.08.0 ([gvm-libs](https://github.com/greenbone/gvm-libs/tree/gvm-libs-20.08) component)
-* PostgreSQL database >= 9.6 (Debian packages: libpq-dev postgresql-server-dev-11)
-* pkg-config (Debian package: pkg-config)
-* libical >= 1.0.0 (Debian package: libical-dev)
-* xsltproc (Debian package: xsltproc)
-* gpgme
 
-Install these prerequisites on Debian GNU/Linux 'Buster' 10:
+* cJSON >= 1.7.14
+* cmake >= 3.0
+* GCC
+* glib-2.0 >= 2.42
+* gnutls >= 3.2.15
+* gpgme
+* [gvm-libs](https://github.com/greenbone/gvm-libs/) >= 22.12
+* libical
```

Comment:
Any reason you dropped the version?

Reply:
No, it was just a mistake. Thanks for catching it 👍🏻